src/main/java/Send_Message/RoomMessageHistoryDBAccess.java

```diff
@@ -0,0 +1,21 @@
+package send_message;
+
+
+import java.util.List;
+import java.util.ArrayList;
+public class RoomMessageHistoryDBAccess implements RoomMessageHistoryDBGateway{
+    private List<MessageDBModel> messageList;
+    public RoomMessageHistoryDBAccess() {
+        // Initialize the messageList in the constructor
+        messageList = new ArrayList<>();
+    }
+    @Override
+    public List<MessageDBModel> loadMessages() {
+        return messageList;
+    }
+
+    @Override
+    public void addMessage(MessageDBModel message) {
+        messageList.add(message);
+    }
+}
```

Comment:
The class directly creates an instance of ArrayList in its constructor. This ties the class to a specific implementation, violating the Dependency Inversion Principle.